packages/components/src/icon-button/icon-button.tsx

```diff
@@ -24,7 +25,27 @@ const Base = styled(Stack, {
     backgroundColor: '$neutral-20',
   },
 
+  pressStyle: {
+    backgroundColor: '$neutral-20',
+  },
+
   variants: {
+    noBackground: {
```

Comment:
This is good enough for now. But in general, I want to follow the components defined as they are in Figma. For `Composer` they don't use IconButton, but different one ([link](https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n/Design-System-for-Web?node-id=10466%3A128996&t=nTM2fEIqcXy97BKt-4)).